exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -50,12 +49,12 @@ type PrwExporter struct {
 func NewPrwExporter(namespace string, endpoint string, client *http.Client) (*PrwExporter, error) {
 
 	if client == nil {
-		return nil, errors.New("http client cannot be nil")
```

Comment:
Just out of curiosity, why these error changes? and just as a heads up these may have merge conflicts because of a PR I recently merged upstream: https://github.com/open-telemetry/opentelemetry-collector/pull/2053

Reply:
It was mainly because Yang made the change in his PR. Considering Bogdan's change, I'll change it to return permanent errors.